packages/dashboard/test/dashboard-layout.test.ts

```diff
@@ -11,19 +12,30 @@ import {
   getScrollingContainer,
   onceResized,
   setColspan,
-  setGap,
   setMaximumColumnCount,
   setMaximumColumnWidth,
   setMinimumColumnWidth,
   setMinimumRowHeight,
   setRowspan,
+  setSpacing,
 } from './helpers.js';
 
+const [defaultSpacing] = (() => {
+  const div = document.createElement('div');
+  document.body.appendChild(div);
+  div.style.width = '1rem';
+  const width = getComputedStyle(div).width;
+  div.remove();
+  return [parseFloat(width)];
+})();
+
+// TODO: Fix the default minimum column width
```

Comment:
Is this something to be addressed later?

Reply:
Addressed now with the current known default